demo/c/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+cmake_minimum_required(VERSION 3.4)
+project(porcupine_demo_c)
+
+set(CMAKE_C_STANDARD 99)
+
+list(APPEND COMMON_LIBS dl)
+list(APPEND MIC_LIBS pthread m)
+
+list(APPEND INCLUDES "${PROJECT_SOURCE_DIR}/../../include")
+
+include_directories(${INCLUDES})
+
+add_compile_options("-O3;")
```

Comment:
i think you can use `set(CMAKE_BUILD_TYPE Release)`. It automatically set the best optimization variables for us.